binderhub/templates/page.html

```diff
@@ -16,6 +16,9 @@
   <link id="favicon" rel="shortcut icon" type="image/png" href="{{static_url("favicon.ico")}}" />
   <link href="{{static_url("dist/styles.css")}}" rel="stylesheet"></link>
 
+  <style>
```

Comment:
Jinja has [control structures](https://jinja.palletsprojects.com/en/stable/templates/#list-of-control-structures). Can we add a [if-clause](https://jinja.palletsprojects.com/en/stable/templates/#if) here?

```html
{% if extra_css %}
<style>
  {{ extra_css }}
</style>
```
```

Reply:
Good idea, I've fixed this